pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/guide.en-gb.md

```diff
@@ -50,17 +53,55 @@ $ openstack loadbalancer listener stats show <listener id>
 
 ### Monitoring with Prometheus
 
-To add a Prometheus endpoint on a Public Cloud Load Balancer, create a listener with a `PROMETHEUS` special protocol. This will enable the endpoint `/metrics` on the listener. This listener type provides the same features as the "regular" listeners, such as `allowed_cidrs`, but does not support attaching pools or L7 policies. All metrics will be identified by the Octavia object ID (UUID) of the resources.
+To add a Prometheus endpoint on a Public Cloud Load Balancer, create a listener with a `PROMETHEUS` special protocol. This will enable the  `/metrics` endpoint which by default listens on all the interfaces from your load balancer. **This means that if your load balancer is using a Floating IP, by default the `/metrics` endpoints will be opened to the internet**. To avoid that, we advise to set the `allowed_cidrs`` option to protect your metrics, for example with the CIDR of the subnet where the load balancer is spawned.
```

Comment:
```suggestion
To add a Prometheus endpoint on a Public Cloud Load Balancer, create a listener using the `PROMETHEUS` protocol. This will enable the  `/metrics` endpoint which by default listens on all the interfaces from your load balancer. **This means that if your load balancer is using a Floating IP, by default the `/metrics` endpoint will be opened to the internet**. To avoid that, we advise to set the `allowed_cidrs` option to protect your metrics, for example with the CIDR of the subnet where the load balancer is spawned.
```